src/components/atoms/ModalHeader/ModalHeader.js

```diff
@@ -47,15 +47,15 @@ export default class ModalHeader extends HTMLElement {
 
     const btnDark = this.getAttribute('data-button-dark');
 
-    const extraClasses = this.getAttribute('data-extra-classes');
+    const extraClasses = this.getAttribute('extra-classes');
```

Comment:
This shouldn't be changed. It's not a button attribute. This is an attribute on ModelHeader.